chromadb/test/test_api.py

```diff
@@ -1428,8 +1428,19 @@ def test_invalid_embeddings(api):
     assert "embedding" in str(e.value)
 
 
+def test_system_info(api_obs) -> None:
+    if not isinstance(api_obs, chromadb.api.fastapi.FastAPI):
```

Comment:
do we know what our api_obs is in testing? if not why not? i'm a little confused by a test which has two totally disjoint paths

Reply:
We split env info on the client and server; we very well know what api_obs provides, but depending on what it gives, we branch and check the client and the server props.